build/stack-local.yaml

```diff
@@ -0,0 +1,52 @@
+ghc-options:
+    $locals:
+        -DFastCompilation
+        -fenable-rewrite-rules
+        -threaded
+        -funbox-strict-fields
+        -O0
+        -fconstraint-solver-iterations=100
+    $everything:
+        -DFastCompilation
+        -fenable-rewrite-rules
+        -threaded
+        -funbox-strict-fields
+        -O0
+        -fconstraint-solver-iterations=100
```

Comment:
Why these particular options?

I think that whole `$everything` section should be removed, otherwise you will build the whole LTS with `-O0` making it very slow. If any option here is needed to build, I would like to know why.